src/metadata.rs

```diff
@@ -85,10 +71,8 @@ fn convert_exif_date_time_to_chrono_date_time_fixed_offset(
 
     match maybe_date {
         chrono::LocalResult::Single(date) => Ok(date),
-        chrono::LocalResult::Ambiguous(_, _) => Err(MetadataError::from_str("Ambiguous date")),
-        chrono::LocalResult::None => Err(MetadataError {
-            message: "Invalid date".to_string(),
-        }),
+        chrono::LocalResult::Ambiguous(_, _) => Err(MetadataError::AmbiguousDate),
+        chrono::LocalResult::None => Err(MetadataError::InvalidDate),
```

Comment:
With this change we do not give back the invalid date in the error. Is it possible to add this?

Reply:
I updated code to pass the date, but thought that maybe it would also be useful to print the faulty picture? Error then in current updated implementation would look like: 
`Could not parse the date: 2019-02-10 13:11:61 found in this path: /path/to/img.jpg`

Reply:
Great idea!